dulwich/porcelain.py

```diff
@@ -1753,6 +1755,61 @@ def update_head(repo, target, detached=False, new_branch=None):
             r.refs.set_symbolic_ref(b"HEAD", to_set)
 
 
+def reset_file(repo, file_path: str , target: bytes = b'HEAD'):
```

Comment:
extra space after "str"

Reply:
If I remove extra space after "str", the flake8 will output: `missing whitespace after ':'flake8(E231)`

Reply:
presumably if you delete the space before rather than after "str"?